generate.py

```diff
@@ -42,6 +44,18 @@ def write_wav(waveform, sample_rate, filename):
     librosa.output.write_wav(filename, y, sample_rate)
     print('Updated wav file at {}'.format(filename))
 
+def create_seed(filename, sample_rate, quantization_channels, percent=50):
+    audio = librosa.load(filename, sr=sample_rate, mono=True)
+    audio = audio[0]
+
+    mu = quantization_channels - 1
+    magnitude = np.log(1 + mu * np.abs(audio)) / np.log(1. + mu)
+    signal = np.sign(audio) * magnitude
+    quantized = (signal + 1) / 2 * mu
+    quantized = quantized.astype(np.int32)
+
+    cut_index = int(len(quantized) * (percent / 100))
```

Comment:
Maybe we need
```python
from __future__ import division
```
in the head of this file, as well as f6c79ee. Because `percent  / 100` can be zero on Python2.